core/src/db/engine/ExecutionEngineImpl.cpp

```diff
@@ -771,8 +808,8 @@ ExecutionEngineImpl::Search(int64_t n, const uint8_t* data, int64_t k, const mil
     auto result = index_->Query(dataset, conf, (blacklist_ ? blacklist_->bitset_ : nullptr));
     rc.RecordSection("query done");
 
-    LOG_ENGINE_DEBUG_ << LogOut("[%s][%ld] get %ld uids from index %s", "search", 0, index_->GetUids()->size(),
-                                location_.c_str());
+//    LOG_ENGINE_DEBUG_ << LogOut("[%s][%ld] get %ld uids from index %s", "search", 0, index_->GetUids()->size(),   ------ > REMOVED by eitan , need to be uncomment
+//                                location_.c_str());
     CopyResult(result, n * k, distances, labels);
```

Comment:
Did you commit it by mistake?

Reply:
yes , it was submitted by mistake . sorry for that.